ecpy_hqc_legacy/instruments/drivers/visa_tools.py

```diff
@@ -120,6 +121,7 @@ def reopen_connection(self):
         as previously.
 
         """
+        self._driver.timeout = 100
```

Comment:
Why do you alter this setting ? You are overriding the user value.

Reply:
leftovers from debugging the SGS instrument. I put it back